Tor_KillTor_code.js

```diff
@@ -26,5 +26,5 @@ PROCESS_ERROR_OUTPUT = base64_decode(split[1])
 {
 PROCESS_STANDART_OUTPUT = _result()
 }
-sleep(1000)!
```

Comment:
?

Reply:
It's for waiting more time. I am thinking that the remove file is executing early.